main.go

```diff
@@ -151,6 +156,11 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
 
+	if err := logsv1.ValidateAndApply(logOptions, nil); err != nil {
```

Comment:
there is already a function `validateFlags` above to validate the flags, wondering if you wanna move in that function?

Reply:
Updated as suggested